src/main/java/gift/product/controller/OrderController.java

```diff
@@ -0,0 +1,26 @@
+package gift.product.controller;
+
+
+import gift.product.commons.annotations.Authenticated;
+import gift.product.dto.CreateOrderRequest;
+import gift.product.service.OrderService;
+import jakarta.validation.Valid;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+
+	private final OrderService orderService;
+	public OrderController(OrderService orderService) {
```

Comment:
```suggestion
	private final OrderService orderService;
	
	public OrderController(OrderService orderService) {
```

https://google.github.io/styleguide/javaguide.html#s4.6.1-vertical-whitespace
스타일 가이드에 맞춰서 개발해보는 것도 좋은 방법 중 하나 입니다 :)

Reply:
넵 다음부턴 저런 부분에 좀 더 유의하겠습니다. 친구들끼리 혹은 개인으로 프로젝트를 할 땐 이런부분을 거의 신경쓰지 않았던 것 같은데 현업, 리뷰 이런 부분에서는 중요하게 여겨지는 부분이란걸 카테캠하면서 여러번 느끼는것같습니다